imls-frontend/src/views/PageLibrary.vue

```diff
@@ -62,14 +97,40 @@ export default {
     },
     startOfWeekInISO() {
       return formatISO(startOfWeek(parseISO(this.selectedDate)), { representation: 'date' })
+    },
+    libraryName() {
+      if (this.fetchedLibraryData && this.fetchedLibraryData.libname )  return this.fetchedLibraryData.libname;
+      return "Library " + this.id
+    },
+    breadcrumbs () {
+      if ( this.fetchedLibraryData == null ) return []
+      return [
+         { 
+          name: "All States",
+          link: "/" 
+        },
+        { 
+          name: this.store.states[this.fetchedLibraryData.stabr],
+          link: `/state/${this.fetchedLibraryData.stabr}` 
+        },
+        {
+          name: this.libraryName
+        }
+      ]
     }
   }
 };
 </script>
 
 <template>
   <div>
-    <h1>Library {{ id }}</h1>
+    <USWDSBreadcrumb :crumbs=breadcrumbs />
+    <h1>{{ libraryName }}</h1>
+    <div v-if="fetchedLibraryData !== null">
+      <h2>{{ formatFSCSandSequence(fetchedLibraryData.fscskey, fetchedLibraryData.fscs_seq) }}</h2>
```

Comment:
to reviewers -- in a bunch of places, i'm concatenating the fscskey and sequence on the front end for display, because this isn't consistent across the records i was seeing for libraries. Ideally, the returned response for a library would contain some unique designator already, and not need the front end to format it. At your convenience (after the demo) could we look at what the library query is returning and see if we can get to a unique id for each library that follows some sort of pattern?

Reply:
We should ticket this for future work; fixing our lookup data is beyond scope right now, but it needs to be not lost. Could you please issue ticket it, since you have the problem in your head?

Reply:
@jadudm @anagradova you two are likely closest to this work. here's the requested issue: https://github.com/IMLS/estimating-wifi/issues/312